commit f7e941785e811e305445d5544cffcfe889a3abc2
Author: Rafael Roquetto <rafael.roquetto.qnx@kdab.com>
Date:   Wed Dec 5 21:43:01 2012 -0200

    configure.exe: fix -inotify option
    
    cherry-picked from qt5 stable 675676aba1e71410fa6cfca430ab9bd64ebdcdaa
    
    Change-Id: I94e8ce037de7df50511be2471db842cca726ccef
    Reviewed-by: Oswald Buddenhagen <oswald.buddenhagen@digia.com>

diff --git a/tools/configure/configureapp.cpp b/tools/configure/configureapp.cpp
index 63ff44f..80cf78b 100644
--- a/tools/configure/configureapp.cpp
+++ b/tools/configure/configureapp.cpp
@@ -3023,6 +3023,9 @@ void Configure::generateOutputVars()
     else if (dictionary["QT_ICONV"] == "gnu")
         qtConfig += "gnu-libiconv";
 
+    if (dictionary["QT_INOTIFY"] == "yes")
+        qtConfig += "inotify";
+
     if (dictionary["NEON"] == "yes")
         qtConfig += "neon";
 
